### PR TITLE
docs: add missing leading slashes in help patterns

### PR DESCRIPTION
**Pull request by remyabel**
Fixes: https://github.com/borgbackup/borg/issues/5857

### CODE REVIEW

File: src/borg/archiver.py [hunk 1]
```diff
@@ -2348,8 +2348,8 @@ def do_break_lock(self, args, repository):
             /home/*/junk
             *.tmp
             fm:aa:something/*
-            re:^home/[^/]\\.tmp/
-            sh:home/*/.thumbnails
+            re:^/home/[^/]+\\.tmp/
+            sh:/home/*/.thumbnails
```
Comment (ThomasWaldmann) [inline src/borg/archiver.py:2352]: btw, for this PR it is sufficient to change these 2 lines.

did you do the other changes manually or did you run python setup.py build_man/build_usage?
it is not needed to do that, i run these at release time.

Comment (remyabel) [reply src/borg/archiver.py:2352]: I did it manually. Would you like me to revert the changes?

Comment (ThomasWaldmann) [reply src/borg/archiver.py:2352]: yeah, reverting the 2 unneeded changes would be good. thanks!

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**